sktime/param_est/seasonality.py

```diff
@@ -2,10 +2,11 @@
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
 """Parameter estimators for seasonality."""
 
-__author__ = ["fkiraly"]
-__all__ = ["SeasonalityACF"]
+__author__ = ["fkiraly", "blazingbhavneek"]
+__all__ = ["SeasonalityACF", "Seasonality"]
 
 import numpy as np
+from seasonal.periodogram import periodogram
```

Comment:
Soft imports should not be in the scope of the file but should rather be limited to the method(s) that use it. Take a look at tensorflow/keras in the DL submodule for an example of how to do that (It involves simply importing it within the method where this package is used.)

The reason behind this is simple - being a soft dependency, one might not have installed it but may still want to use a function/object (and this function/object should not be making use of this dependency). A good software design will not throw an error for an import which isn't supposed to be used in the code execution.

Reply:
Done!